\documentclass[12pt, English]{article}
\usepackage{geometry}
\geometry{verbose,letterpaper,tmargin=2.54cm,bmargin=2.54cm,lmargin=2.54cm,rmargin=2.54cm}
%\usepackage{apacite}
\usepackage[natbibapa]{apacite}
%\renewcommand{\APACrefYearMonthDay}[3]{\APACrefYear{#1} 
%  \cite<pre>[post]{key}
\usepackage{amsfonts}
\usepackage{amssymb}
\usepackage{amstext}
\usepackage{amsmath}
\usepackage{comment}
\usepackage{enumerate}
\usepackage{graphicx}
\usepackage{boxedminipage}
\usepackage{oldgerm}
\usepackage{setspace}
\usepackage{booktabs}
\usepackage{wrapfig}
\usepackage{indentfirst}
\usepackage{mathptmx}
\usepackage{setspace}
\usepackage{lscape} 
\usepackage[mathlines]{lineno}
\renewcommand{\ttdefault}{mathptmx}
\usepackage[latin9]{inputenc}
\usepackage{bm}
\usepackage{relsize}
\usepackage[normalem]{ulem} %%
%\linenumbers
\usepackage{bm}
\setlength{\parindent}{20pt}
\usepackage[labelformat=empty,labelsep=none]{caption}
\usepackage[singlelinecheck=false]{caption}
\usepackage{color}
\def\RED#1{\textcolor{red}{#1}} %% 
\def\BLUE#1{\textcolor{blue}{#1}} %% 

% https://onlinelibrary.wiley.com/page/journal/17550998/homepage/forauthors.html

\begin{document}

\title{Estimating contemporary migration numbers of adults for iteroparous species based on kinship relationships found between populations}
\author{
Tetsuya Akita$^{a}$\thanks{
\emph{Corresponding author}:
    Fisheries Resources Institute, Fisheries Research and Education Agency, Kanagawa 236-8648, Japan.
    E-mail: akitatetsuya1981@affrc.go.jp} \\ 
   \small $^{a}$Fisheries Resources Institute, Fisheries Research and Education Agency, \\
   \small Kanagawa, 236-8648, Japan.
   }
\date{Manuscript Intended for \emph{Molecular Ecology Resources}, \today}% Manuscript Intended for \emph{Molecular Ecology and Resources}
\maketitle

\begin{abstract}
In this paper, we describe the development of estimators for the contemporary migration number and rate of adults of iteroparous species between two populations. The proposed estimators are based on known half-sibling (HS) and/or parent--offspring (PO) relationships observed between the populations across breeding seasons. The rationale is that the HS and PO pairs exhibit information about the occurrence frequency of parental movements during the breeding interval. The proposed method allows for variance in the average number of offspring per parent both within and between populations. In addition, coupled with the PO pairs found within the population, the estimators can be obtained from only genetic data. Generally, a sample size representing the square root of the population size is required to obtain meaningful information regarding migration. We describe a detailed evaluation of the performance of the estimators by running an individual-based model, and the results provide guidance regarding sample sizes to ensure the required accuracy and precision. Furthermore, given that there are few effective methods to estimate adult movement (especially when populations cannot be genetically distinct), we discuss the usefulness of this proposed kinship assignment method in terms of conservation biology and wildlife management.  
\\
\end{abstract}
\textbf{Key words:} dispersion; kinship assignment; reproductive potential; close-kin mark-recapture\\
\textbf{Running head:} Estimating migration numbers using kinship pairs\\
\textbf{Subject Area:} Molecular and Statistical Advances\\

\pagebreak
\begin{spacing}{1.9}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%FIGURE S1%%%
\newcommand{\NeNbias}{S1}
%%%%%%%%%%%

%%%FIGURE S2%%%
\newcommand{\Nebias}{S2}
%%%%%%%%%%%

%%%FIGURE S3%%%
\newcommand{\Ninvbias}{S3}
%%%%%%%%%%%

%%%FIGURE S4%%%
\newcommand{\NeNcv}{S4}
%%%%%%%%%%%

%%%FIGURE S5%%%
\newcommand{\Necv}{S5}
%%%%%%%%%%%

%%%FIGURE S6%%%
\newcommand{\Ninvcv}{S6}
%%%%%%%%%%%

\section{INTRODUCTION}

% 1: Estimating contemporary migration is important for both ecologically and evolutionally reasons

In conservation and wildlife management, estimating current ecological information is essential for monitoring population levels and proposing effective management strategies. The contemporary migration rate, i.e., the recent rate of movement of individuals or gametes between predefined populations, is a critical component when determining the current degree of gene flow and synchrony among the population \cite[]{Waples_2006_MolEcol, https://doi.org/10.1111/j.1365-294X.2010.04688.x}. Therefore, estimating the contemporary migration rate provides information about the degree of genetic differentiation and demographic dependency, which can delineate evolutionarily significant units and identify appropriate management units \cite[]{moritz1994defining, PALSBOLL200711}. In addition, the availability of migration rate can link to underlying population dynamics, allowing for more precise and flexible evaluations of management practices, e.g., stock assessment in fisheries \cite[]{hampton2001spatially} and invasive species control \cite[]{doi:10.1146/annurev.ecolsys.32.081501.114037}. 

% 2: Mark recapture method can handle ecological scale but may miss the small migration rate

Essentially, there are two approaches to estimate the contemporary migration rate. The first approach involves using the mark-recapture (MR) method, which estimates the migration rate along with other population parameters \cite[]{kery2011bayesian, Thorson_2021}. Here, the rational is that, for example, conventional tags provide release and recovery location information for known release and recovery dates, which generates a movement fraction matrix among strata per given time interval. This direct method is relatively easy to interpret; however, it is generally hampered by several uncertainties, e.g., tag loss, tagging-related mortality, and time-varying reporting rates \cite[]{hilborn2013quantitative}. In addition, the MR method is generally limited in practice to situations where collection of a sufficient number of adult samples at multiple sites is possible over a short period. 

% 3: Population genetics method can handle evolutionally scale but may miss the large migration rate

The second approach involves using numerous genetic markers to assign individuals to source populations, thereby allowing the inference of recent migration \cite[since][]{https://doi.org/10.1111/j.1365-294X.1995.tb00227.x}. The previously developed model of \cite{Wilson_2003}, which is one of the population assignment methods and implemented in a software `BayesAss', can output the point estimate as an element of the current migration matrix. However, according to several papers \cite[]{https://doi.org/10.1111/j.1365-294X.2007.03218.x, https://doi.org/10.1111/mec.12806}, the accuracy of the migration rates estimated by this model is valid only when populations are highly differentiated (e.g., $F_{\rm ST}\ge0.05$), which suggests that it may be difficult to infer the degree of demographic dependency among populations because demographic independency is realized even if there are many migrants. 

% 4: Wang 2014 can handle ecological scale with realistic situations but limited to offspring movement

Parental assignments, which are based on genetic markers and are frequently used to complement the population assignment method in ecological studies, also provide information about the current level of migration. In contrast to the population assignment method, the parental assignment method does not require population differentiation \cite[]{https://doi.org/10.1111/mec.12806}; however, the current methodology focuses on estimating dispersal kernels and, to estimate the migration rate, requires the assumption that adults are not migrant, e.g., pollen/seed dispersal in plants \cite[]{doi:10.1080/07352689.2010.481167} or larval drift in marine animals \cite[]{https://doi.org/10.1111/eva.12288}. Thus, this method may not consider the adult movement of iteroparous species, i.e., multiple reproductive cycles during the lifetime, which may change the spawning ground for each breeding season. 

% 5: CKMR can cover the range that Mark recapture can handle with more realistic situations

Close-kin mark-recapture (CKMR) is a recently developed method to estimate adult population size along with life-history parameters that utilizes the known kinship information in a sample \cite[][and references therein]{https://doi.org/10.1111/faf.12615}. In the CKMR method, the presence of a kinship pair in the sample is analogous to the recapture of a marked individual in the MR method. Note that kinship pairs in the sample are less likely to be observed in larger populations; thus, the number of kinship pairs may reflect the number of adults in the population \cite[]{bravington2016close}. While recent studies of CKMR argued that availability of kinship information to identify metapopulation structure \cite[]{https://doi.org/10.1002/ece3.6296, 10.1093/icesjms/fsac002}, CKMR-based migration estimation has not been well developed or examined extensively in a simulation study. 

% 6: In this paper, ~

Thus, in this paper, we propose a method to estimate the contemporary migration number or rate of adults between two predefined populations in iteroparous species, where the migration direction is specified. Under the assumption that kinships are genetically detected without error, the proposed method is based on the numbers of half-sibling (HS) and parent--offspring (PO) pairs in a sample. Here, sampling can be either invasive or noninvasive and is completed at two breeding seasons; sampling offspring (young-of-year individuals) and parents likely share a PO relationship with the offspring sampled in one population at the first breeding season and in the other population at the second breeding season. The rationale for estimating the migration number or rate is that the number of HS and PO pairs found between populations contains information about the frequency of parental movements during the breeding interval. Our model explicitly incorporates reproductive variation within and between populations, thereby making it possible to target a species whose fertility is affected by environmental differences between populations. First, we explain the modeling assumption and sampling scheme. Then, we analytically determine the estimators of the contemporary migration number or rate of adults, which are based on the numbers of HS and/or PO pairs. Finally, by running an individual-based model, we investigate the performance of the estimator and provide a guide for a sample size. Note that the proposed modeling framework can be applied to diverse animal species; however, the description of the model focuses on fish species, which are presently the best target candidate for the proposed method.

\section{Theory}\label{sec2}
Here, we present the theoretical foundation of estimating a migrant number and migration rate of iteroparous species using HS or PO pairs found between two populations with different sample timing. Note that the estimators can hold under flexible assumptions for reproduction ({\bf Appendix 2}); however, here, we assume a relatively simple case. The main symbols used in this paper are summarized in {\bf Table \ref{symbols}}. 

%%%%%%%%%%%%%%
\begin{center}
\fbox{Table \ref{symbols} is here. }
\end{center}
%%%%%%%%%%%%%%

\subsection{Hypothetical population}

Assume that we have a set of populations 1 and 2, where random mating occurs within each population and parents can move to the other population after the reproductive season ends. Without loss of generality, we consider the movement of parents from population 1 to population 2, and we focus on estimating the migration number or rate. In this framework, we demonstrate that considering two reproductive seasons (hereafter referred to as "the first year" and "the second year") is sufficient to estimate the migration number or rate. {\bf Figure \ref{cartoon}a} shows a schematic representation of the kinship relationships and parent movements. 

Assume that there are $N_1$ parents in population 1 at the beginning of the first year. Each parent produces a number of offspring that is governed by the parent's reproductive potential, which is denoted by $\lambda_{i,1}$ ($i=1,2,\ldots,N_1$). After the reproductive season, some parents begin to move toward population 2. Here $M$ survived migrants arrive in population 2; thus, there are $N_2$ parents in population 2 at the beginning of the second year ($M \le N_1$ and $M \le N_2$). Similar to population 1, $N_2$ parents produce a number of offspring governed by $\lambda_{j,2}$ ($j=1,2,\ldots,N_2$). Note that reproductive potential is determined by several factors. Additional details about reproductive potential can be found in {\bf Appendix 1}. Theoretically, the number of offspring for each parent is set as a random variable with mean $\lambda_{i,1}$ or $\lambda_{j,2}$. 

%%%%%%%%%%%%%%
\begin{center}
\fbox{Figure \ref{cartoon} is here. }
\end{center}
%%%%%%%%%%%%%%

\subsection{Sampling}

To estimate the migration number or rate, we utilize the number of HS and/or PO pairs observed in a sample. In both populations, $n_{\rm O, 1}$ and $n_{\rm O, 2}$ offspring are sampled randomly in the first and second year, respectively, which are identified as young-of-year individuals without error. In addtion, $n_{\rm P, 1}$ and $n_{\rm P, 2}$ parents are sampled randomly immediately after the end of the reproductive season in the first and second years, respectively. For mathematical tractability, the parents must survive the reproductive season; therefore, both the mother and father of a give offspring have the potential to be sampled in the same year. Note that all the four sample types, i.e., $n_{\rm P, 1}$, $n_{\rm P, 2}$, $n_{\rm O, 1}$, and $n_{\rm O, 2}$, are not always required to estimate the migration number or rate. The types of required samples depends on the given situation, which is explained later. Up to subsection 2.6, we focus on only the HS and PO pairs found between populations 1 and 2. To apply the proposed method to the fishery assessment, we assume an invasive sampling procedure, which prevents us from finding a PO pair such that a parent and offspring is sampled in population 1 and 2, respectively. {\bf Figure~\ref{cartoon}b} shows the timeline of the sampling scheme for these hypothetical populations. 

In the example shown in {\bf Fig.~\ref{cartoon}a}, five offspring and four parents are sampled in population 1, four offspring and six parents are sampled in population 2, and two HS pairs and a single PO pair are observed between the two populations 1 and 2. In addition, there are several PO pairs observed within the same population, which are available to estimate parent numbers, which is explained in subsection 2.7. In our modeling framework, if a full-sibling relationship is found, we count it as two HS pairs. The numbers of HS and PO pairs found between the two populations are determined via pairwise comparison of all the sample individuals, i.e., comparisons of $n_{\rm O,1}\times n_{\rm O,2}$ and $n_{\rm O,1}\times n_{\rm P,2}$, respectively. 

\subsection{HS pair-based model}

Here, we consider the probability that two offspring sampled in populations 1 and 2 will share an HS relationship with an arbitrary mother or father, denoted by $\pi_{\rm HS, bet}$. Note that we assume equal reproductive potential among parents (up to subsection 2.7) although this assumption can be relaxed for most of the case (see {\bf Appendix 2} for additional information). Here, $\pi_{\rm HS, bet}$ can be partitioned into three probabilities, i.e., (i) the probability that sampled offspring in population 1 born to a parent that safely arrives in population 2 (hereafter, referred to as a "migrant"); (ii) the probability that sampled offspring in population 2 are born to a migrant; and (iii) the probability that the migrant of a sampled offspring in population 1 and the migrant of a sampled offspring in population 2 are identical. 

To assess these probabilities, recall that there must be two parents of arbitral offspring. The first probability is the sum of the father--offspring and mother--offspring relationship, which can be expressed as $r_{\rm M}M/(r_{\rm S,1}N_{1}) + (1 - r_{\rm M})M/((1-r_{\rm S,1})N_{1})$, where $r_{\rm M}$ and $r_{\rm S,1}$ indicate the sex ratio of the parents in the migrants and in all of the population 1, respectively. Under the assumption that $r_{\rm M} = r_{\rm S,1}$, the probability is simplified to $2M/N_{1}$. Similarly, the second probability is simplified to $2M/N_{2}$, which implies the assumption regarding an equal sex ratio in migrants and others. The third probability corresponds to the probability that two randomly selected migrants are identical, i.e., $1/M$. Taken together, we obtain the following: 
\begin{align}
\pi_{\rm HS,bet} &= \frac{2M}{N_{1}} \frac{2M}{N_{2}} \frac{1}{M} \nonumber\\
&= \frac{4M}{N_{1}N_{2}}.
\label{HS-1}
\end{align}
This form holds under a flexible setting for the reproductive potential, e.g., a situation with random variable of $\lambda_{i,1}$ and $\lambda_{j,2}$ ({\bf Appendix 2}). If all $N_1$ parents safely move to population 2 (i.e., $M = N_1$), $\pi_{\rm HS,bet}$ equals $4/N_2$. In addition, if population 2 comprises only migrants (i.e., $M = N_2$), $\pi_{\rm HS,bet}$ equals $4/N_1$. In extreme cases, these probabilities take similar forms as the HS probability, which is sampled randomly from two different cohorts within a population \cite[]{bravington2016close}. 

Let $H_{\rm HS,bet}$ be the number of HS pairs found in the offspring samples of size $n_{\rm O, 1}$ and $n_{\rm O, 2}$. Assuming the total number of HS pairs in the population is much greater than $H_{\rm HS,bet}$, the distribution is approximated by a binomial form (i.e., $H_{\rm HS,bet} \sim {\rm Binom}[\pi_{\rm HS,bet}, n_{\rm O, 1} n_{\rm O, 2}]$). Thus, the theoretical expectation of $H_{\rm HS,bet}$ is given as follows: 
\begin{align}
\mathbb{E}[H_{\rm HS, bet}] &= \pi_{\rm HS,bet} n_{\rm O, 1} n_{\rm O, 2} \nonumber\\
&= \frac{ 4 n_{\rm O, 1} n_{\rm O, 2} M } { N_{1} N_{2} } \label{HS-2-1}\\
&= \frac{ 4 n_{\rm O, 1} n_{\rm O, 2} m } { N_{2} },
\label{HS-2-2}
\end{align}
where $m$ is the migration rate satisfying $M = mN_{1}$. The observed number of HS pairs in a sample found between the populations 1 and 2 is defined by $\widetilde{H}_{\rm HS, bet}$, and $\mathbb{E}[H_{\rm HS, bet}]$ in Equation \ref{HS-2-1} is replaced by $\widetilde{H}_{\rm HS, bet}$, thereby generating the linear estimator of $M$:

\begin{align}
\widehat{M_1} &= \frac{N_{1} N_{2} \widetilde{H}_{\rm HS, bet} } {4 n_{\rm O, 1} n_{\rm O, 2} }.
\label{HS-3}
\end{align}
In this paper, a ``tilde'' and ``hat'' symbols indicates the observation and estimator of a variable, respectively. Similarly, $\mathbb{E}[H_{\rm HS, bet}]$ in Equation \ref{HS-2-2} is replaced by $\widetilde{H}_{\rm HS, bet}$, which generates the linear estimator of $m$:

\begin{align}
\widehat{m_1} &= \frac{N_{2} \widetilde{H}_{\rm HS, bet} } {4 n_{\rm O, 1} n_{\rm O, 2} }
\label{HS-4}
\end{align}
The subscripts associated with these estimators indicate the numbering of the proposed estimators, which is summarized in Table \ref{parameter}. 

%%%%%%%%%%%%%%
\begin{center}
\fbox{Table \ref{parameter} is here. }
\end{center}
%%%%%%%%%%%%%%

%KOKOMADE enago%%%
\subsection{PO pair-based model}

Next, we consider the probability that offspring sampled in the population 1 and a parent sampled in the population 2 share an PO relationship, denoted by $\pi_{\rm PO, bet}$. $\pi_{\rm PO, bet}$ can be partitioned into the three probabilities: (i) the probability that sampled offspring in the population 1 born to a migrant; (ii) the probability that a migrant is sampled; and (iii) the probability that the migrant of a sampled offspring in the population 1 and the migrant sampled in the population 2 are identical. The first and third probabilities are the same as the one introduced in the previous subsection, that is, $2M/N_{1}$ and $1/M$. The second probability is $M/N_{2}$ by definition. Taken together, we obtain
\begin{align}
\pi_{\rm PO,bet} &= \frac{2M}{N_{1}} \frac{M}{N_{2}} \frac{1}{M} \nonumber\\
&= \frac{2M}{N_{1}N_{2}}.
\label{PO-1}
\end{align}
This form also holds under a flexible setting for the reproductive potential, as noted in {\bf Appendix}. If all $N_1$ parents safely move to the population 2 (i.e., $M = N_1$), $\pi_{\rm PO,bet}$ equals $2/N_2$; meanwhile, if the population 2 consists only of migrants (i.e., $M = N_2$), $\pi_{\rm PO,bet}$ equals $2/N_1$. These probabilities in the extreme cases are similar forms to PO probability that is randomly sampled within a population \cite[]{bravington2016close}. 

Let $H_{\rm PO,bet}$ be the number of PO pairs found in offspring samples of size $n_{\rm O, 1}$ and parent samples of size $n_{\rm P, 2}$. Assuming that the total number of PO pairs in the two populations is much higher than $H_{\rm PO,bet}$, the distribution is approximated by a binomial form (i.e., $H_{\rm PO,bet} \sim {\rm Binom}[\pi_{\rm PO,bet}, n_{\rm O, 1} n_{\rm P, 2}]$); thus, the theoretical expectation of $H_{\rm PO,bet}$ is
\begin{align}
\mathbb{E}[H_{\rm PO, bet}] &= \pi_{\rm PO,bet} n_{\rm O, 1} n_{\rm P, 2} \nonumber\\
&= \frac{ 2 n_{\rm O, 1} n_{\rm P, 2} M } { N_{1} N_{2} } \label{PO-2-1} \\
&= \frac{ 2 n_{\rm O, 1} n_{\rm P, 2} m } { N_{2} }.
\label{PO-2-2}
\end{align}
The observed number of PO pairs in a sample is defined by $\widetilde{H}_{\rm PO, bet}$, and $\mathbb{E}[H_{\rm PO, bet}]$ in Equations \ref{PO-2-1} and \ref{PO-2-2} are replaced by $\widetilde{H}_{\rm PO, bet}$, generating the linear estimator of $M$ and $m$:
\begin{align}
\widehat{M_2} &= \frac{N_{1} N_{2} \widetilde{H}_{\rm PO,bet} } {2 n_{\rm O, 1} n_{\rm P, 2} },
\label{PO-3}
\end{align}
and
\begin{align}
\widehat{m_2} &= \frac{N_{2} \widetilde{H}_{\rm PO,bet} } {2 n_{\rm O, 1} n_{\rm P, 2} }.
\label{PO-4}
\end{align}

\subsection{Required sample size}

Proposed estimators are based on the observed number of kinship pairs. Their expected number is linearly determined by the number of pairwise comparison (Eqs.~\ref{HS-2-1}-\ref{HS-2-2} and \ref{PO-2-1}-\ref{PO-2-2}), providing guidance for a sample size to ensure the condition that at least one or more kinship pairs can be found. The conditions are given by
\begin{align}
n_{\rm O, 1} n_{\rm O, 2} &> \frac{ N_{2} } { 4m },
\label{RS-1}
\end{align}
and
\begin{align}
n_{\rm O, 1} n_{\rm P, 2} &> \frac{ N_{2} } { 2m }.
\label{RS-2}
\end{align}
Roughly speaking, required sample size to ensure above conditions is that $n>\sqrt{N_2}$, where $n = n_{\rm O, 1}=n_{\rm O, 2}=n_{\rm P, 2}$. It should be noted that very tiny $m$, which dramatically increases the required sample size, produces significant genetic differences between two populations. In such a case, there are several methods to estimate migration rate by population genetics technique; alternatively, we focus the situation of moderate/large migration rate, which leads little genetic differences between two populations. 

Other guidance for the required sample size is obtained by an approximate lower bound on the CV (coefficient of variation) of $M_1$ or $m_1$, defined by $1/\sqrt{H_{\rm HS, bet}}$, which is applied in the context of a classic mark-recapture \cite[]{seber1982estimation} or close-kin mark-recapture \cite[]{bravington2016close}. For example, to achieve a 30\% CV, the target of $H_{\rm HS, bet}$ is more than 10, providing the required sample size when population parameters are given (e.g., $N_1$, $N_2$ and $M$). Similarly, the CV of $M_2$ or $m_2$, defined by $1/\sqrt{H_{\rm PO, bet}}$, provides the required sample size for estimating $M_2$ or $m_2$. 

\subsection{Efficient use of kinship pairs found between populations}

When the both HS pairs and PO pairs, which are found between populations, are simultaneously available, we can obtain efficient estimators by combined $\widehat{M_1}$ and $\widehat{M_2}$ for migration number: 
\begin{align}
\widehat{M_3} &= \frac{ N_{1} N_{2} \left(\widetilde{H}_{\rm HS,bet} + \widetilde{H}_{\rm PO,bet}\right) } {2n_{\rm O, 1} \left(2 n_{\rm O, 2} + n_{\rm P, 2}\right) },
\label{AP-1}
\end{align}
and by combining $\widehat{m_1}$ and $\widehat{m_2}$ for migration rate: 
\begin{align}
\widehat{m_3} &= \frac{ N_{2} \left(\widetilde{H}_{\rm HS,bet} + \widetilde{H}_{\rm PO,bet}\right) } {2n_{\rm O, 1} \left(2 n_{\rm O, 2} + n_{\rm P, 2}\right)}.
\label{AP-2}
\end{align}
For those estimators, $\widetilde{H}_{\rm HS,bet}$ and $\widetilde{H}_{\rm PO,bet}$ is weighted by a sample size $n_{\rm O, 2}$ and $n_{\rm P, 2}$, respectively. 

\subsection{Estimation of parent number by PO pairs found within a population}

So far, we provide the formulation of the estimators, presented in Equations \ref{HS-3}-\ref{HS-4} and \ref{PO-3}-\ref{PO-4}, is a function with parent numbers for each population ($N_1$ and/or $N_2$); in other words, such estimators are available only when those parent number(s) are known. In this subsection, we provide the treatment for estimating the unknown parent numbers by additional usage of PO pairs, producing estimators for migrant number and migration rate that can be obtained only from the genetic data. 

When PO pairs found in offspring samples and parent samples both from the population 2 in the second year are available, standard estimator of parental number can be obtained \cite[]{bravington2016close}, given by

\begin{align}
\widehat{N_2} &= \frac{ 2 n_{\rm O, 2} n_{\rm P, 2}  }{\widetilde{H}_{\rm PO, 2} + 1} ,
\label{AP-3}
\end{align}
where $\widetilde{H}_{\rm PO, 2}$ is the observed number of PO pairs found in offspring samples of size $n_{\rm O, 2}$ and parent samples of size $n_{\rm P, 2}$. The term ``$+1$'' reduces the bias especially when $\widetilde{H}_{\rm PO, 2}$ is small \cite[e.g.,][]{ecolevol2021p}, where a similar derivation of this bias correction is provided in \cite{Akita_2019}. By replacing $N_2$ by $\widehat{N_2}$ in Eqs.~\ref{AP-1} and \ref{AP-2}, we can obtain the following estimators: 

\begin{align}
\widehat{M_4} &=  \frac{ n_{\rm O, 2} n_{\rm P, 2} N_{1} \left(\widetilde{H}_{\rm HS,bet} + \widetilde{H}_{\rm PO,bet}\right) } {n_{\rm O, 1} \left(2 n_{\rm O, 2} + n_{\rm P, 2}\right) \left(\widetilde{H}_{\rm PO, 2} + 1\right)},
\label{AP-4}
\end{align}
and
\begin{align}
\widehat{m_4} &=  \frac{ n_{\rm O, 2} n_{\rm P, 2}\left(\widetilde{H}_{\rm HS,bet} + \widetilde{H}_{\rm PO,bet}\right)} {n_{\rm O, 1} \left(2 n_{\rm O, 2} + n_{\rm P, 2}\right) \left(\widetilde{H}_{\rm PO, 2} + 1\right)}.
\label{AP-5}
\end{align}
While estimator of the migrate number (Equation~\ref{AP-4}) requires the (unknown) parent number in the population 1 ($N_{1}$), estimator of the migration rate (Equation~\ref{AP-5}) can be obtained only the observed kinship pairs. 

Similar to estimating $N_2$, when PO pairs found in offspring samples and parent samples both from the population 1 in the first year are also available, we can obtain the estimator of $N_1$, written by
\begin{align}
\widehat{N_1} &= \frac{ 2 n_{\rm O, 1} n_{\rm P, 1} }{\widetilde{H}_{\rm PO, 1} +1},
\label{AP-6}
\end{align}
where $\widetilde{H}_{\rm PO, 1}$ is the observed number of PO pairs found in offspring samples of size $n_{\rm O, 1}$ and parent samples of size $n_{\rm P, 1}$. By replacing $N_1$ by $\widehat{N_1}$ in Eq.~\ref{AP-4}, we can obtain the estimator of migration number that can be calculated only from genetic data, given by
\begin{align}
\widehat{M_5} &=  \frac{ 2 n_{\rm O, 1} n_{\rm P, 1} n_{\rm O, 2} n_{\rm P, 2} \left(\widetilde{H}_{\rm HS,bet} + \widetilde{H}_{\rm PO,bet}\right)} {n_{\rm O, 1} \left(2 n_{\rm O, 2} + n_{\rm P, 2}\right) \left(\widetilde{H}_{\rm PO, 1} + 1\right) \left(\widetilde{H}_{\rm PO, 2} + 1\right)}.
\label{AP-7}
\end{align}

Table \ref{parameter} summarizes the conditions for estimators appeared in this paper as to whether $N_1$ and/or $N_2$ are known and which type of samples are needed for estimation, and provides required kinship type for calculating the estimator. 

\subsection{Variation in reproductive potential among individuals and populations}

The proposed estimators are derived under the assumption that reproductive potential is the same among individuals. As noted in {\bf Appendix 2}, this assumption can be relaxed and the estimators still hold, which is exemplified in iteroparous species that may show significant variation in reproductive potential among individuals. Furthermore, the estimators even hold for most of the case where mean reproductive potential are different between population 1 and 2, which is exemplified in the situation that the environmental condition for reproductive success varies between populations. The necessary condition is that the migration event does not depends on the degree of reproductive potential (see details in  {\bf Appendix 3}). 

\subsection{Individual-based model}

We developed an individual-based model that tracks kinship relationships to evaluate the estimator's performance. The population structure was assumed to be identical to that in the development of the estimators. The population 1 and 2 comprised $N_1$ and $N_2$ parents with an equal sex ratio, and their offspring number was assumed to follow the geometric distribution with mean ${\bar \lambda_1}$ and ${\bar \lambda_2}$ (i.e., Poisson reproduction with mean $\lambda_{i,1}$ and $\lambda_{j,2}$ which follows the exponential distribution with mean ${\bar \lambda_1}$ and ${\bar \lambda_2}$), respectively. Migrant parents were randomly chosen from the population 1 at the end of the first year. Each offspring retained the parent's ID, making it possible to trace an HS and PO relationship.

Let a parameter set ($N_1$, $N_2$, $M$, ${\bar \lambda_1}$, ${\bar \lambda_2}$, $n_{\rm O,1}$, $n_{\rm O,2}$, $n_{\rm P,1}$, $n_{\rm P,2}$) be given. We simulated a population history and a sampling process, which generates proposed estimators; this process was repeated 1000 times, making it possible to construct the distribution of the estimators for each parameter set. All scripts and documentation for these analyses are available at \\https://github.com/teTUNAakita/CKMRmig.

\section{Results}\label{sec3}

We numerically evaluated the performance of ${\hat M}$s or ${\hat m}$s for the case in which variable reproductive potential among parents. Scaled statistical properties of ${\hat m_1}$,  ${\hat m_2}$,  ${\hat m_3}$ and ${\hat m_4}$ are completely the same to ${\hat M_1}$, ${\hat M_2}$, ${\hat M_3}$ and ${\hat M_4}$, hereafter, we only demonstrate the results of ${\hat M_1}$-${\hat M_5}$. {\bf Figure~\ref{violins}} illustrates the distribution of the relative bias of ${\hat M}$ for limiting cases where parent and offspring sample numbers are identical (i.e., $n=n_{\rm P,1}=n_{\rm P,2}=n_{\rm O,1}= n_{\rm O,2}$) and parent numbers in the two populations are also identical (i.e., $N=N_1=N_2$). Relative bias is calculated by applying outputs of the individual-based model, which is defined as follows: ``(averaged estimator - true value)/true value.'' Full list of parameter sets to evaluate the performance (relative bias and CV of ${\hat M}$s) is summarized in {\bf Table S1} in {\bf Supporting Information}. 
 
%%%%%%%%%%%%%%
\begin{center}
\fbox{Figure \ref{violins} is here. }
\end{center}
%%%%%%%%%%%%%%

First, we evaluated the accuracy of ${\hat M}$s based on their relative bias. As expected, for most of the investigated parameter sets, we observed that their relative bias is less than 5\%, as demonstrated in {\bf Table S1} in {\bf Supporting Information}. Thus, when the assumption that kinships are detected without any error is satisfied, it is reasonable to call ${\hat M}$s (nearly) unbiased estimators. 

Next, we evaluated the precision of ${\hat M}$s based on their CV. {\bf Table S1} in {\bf Supporting Information} demonstrated the value of the CV; meanwhile, the violin plot ({\bf Figure~\ref{violins}}) visually provides the degree of precision. For each estimator ${\hat M}$, the precision increases with increasing of sample size. It should be noted that total sample size depends on the estimator. For example, $n=50$ in Fig.~\ref{violins}, the total sample size of ${\hat M_1}$, ${\hat M_2}$, ${\hat M_3}$, ${\hat M_4}$ and ${\hat M_5}$ is 100 ($=n_{\rm O,1}+n_{\rm O,2}$), 100 ($=n_{\rm O,1}+n_{\rm P,2}$), 150 ($=n_{\rm O,1}+n_{\rm O,2}+n_{\rm P,2}$), 150 ($=n_{\rm O,1}+n_{\rm O,2}+n_{\rm P,2}$) and 200 ($=n_{\rm O,1}+n_{\rm O,2}+n_{\rm P,1}+n_{\rm P,2}$), respectively. 

As sample size or migrants increase, the precision increases and the shape of the distribution asymptotically becomes symmetric ({\bf Figure~\ref{violins}}). This is because an increasing of sample size or migrants is likely to increase the observed number of kinship pairs found between two populations ($\widetilde{H}_{\rm HS,bet}$ or $\widetilde{H}_{\rm PO,bet}$) and decreases the variance of those kinship pair numbers. In addition, the number of PO pair within a population ($\widetilde{H}_{\rm PO,1}$ or $\widetilde{H}_{\rm PO,2}$) also contributes the precision for estimating $N_1$ or $N_1$, respectively, providing ${\hat M_4}$ and ${\hat M_5}$ with a higher precision. 

Our simulator can handle the situation with or without invasive sampling. Invasive sampling potentially affects the level of $m$ since sampled parents in the population 1 have no chance to move to the population 2 and thus those parents cannot be sampled. In this case, $m$ should be defined by $M/(N_1-n_{P,1})$ not by $M/N_1$, although it does not affect the estimator ${\hat M}$s, as demonstrated in {\bf Table S1} ({\bf Supporting Information}). 

Finally, we investigated the situation in which the parental number in the population 1 is much more than in the population 2 (i.e., $N_1 \gg N_2$) and thus migrants are large proportion of $N_2$. In such a setting, e.g., $(N_1,N_2,M)=(10^4,10^3,500)$, we confirmed the similar property of ${\hat M}$s to the situation $N_1 = N_2$, as demonstrated in {\bf Table S1} ({\bf Supporting Information}), suggesting the robustness of ${\hat M}$s in heterogeneous population sizes.   

\section{Discussion}\label{sec4}

We theoretically developed estimators of the contemporary migration number (${\hat M_1}$--${\hat M_5}$) and migration rate (${\hat m_1}$--${\hat m_4}$) of parents between two pre-defined populations in iteroparous species. The proposed estimators are based on known PO relationship and HS relationships observed between and within two populations without any error of kinship assignment. Users can choose the estimator developed for the situation as to whether the parental number of population 1 ($N_1$) and/or that of population 2 ($N_2$) is known ({\bf Table~\ref{parameter}}). The performance of the estimator (accuracy and precision) was quantitatively evaluated by running an individual-based model ({\bf Figure~\ref{violins}}; see also {\bf Table S1} in {\bf Supporting Information}). Our modeling framework utilizes several types of reproductive variations (survived offspring number per parent), including variance of reproductive potential within and between populations, taking account of several situations, including a body-size structure or environmental heterogeneity for reproductive success.

We presented mainly three novel points in this paper. First, we formulated probabilities of kinship pairs randomly chosen between two populations ($\pi_{\rm HS,bet}$ and $\pi_{\rm PO,bet}$), which provide estimators of the migration number or rate. While similar derivations may be found in (nongenetical) the MR method, these are limited to the situation such that sampling can be non-invasive and adult individuals are sampled. Our proposed method can avoid these limitations, where such an advantages is characterized by CKMR method for estimating population sizes. Second, we revealed that the probabilities of kinship pairs are approximately independent of reproductive potential and thus information about it is not needed for the estimation, provided that migration assumes to occur independently of the reproductive potential. This is a useful property of the estimators because it is natural that heterogeneity of reproductive potential within and between populations exits in iteroprous species but its observation is generally difficult except a body size. Third, we demonstrated applicability that the estimation can be obtained only from the genetic data (i.e., ${\hat M_5}$ and ${\hat m_4}$). There are several advantages in using the proposed estimator instead of separately estimating $\pi_{\rm HS,bet}$ (and/or $\pi_{\rm PO,bet}$) (via genetic method) and $N_1$ (and/or $N_2$) (via nongenetical method), including simplified sampling and analyzing designs and availability of unified framework of genetic analyzes for detecting HS and PO pairs (similar discussion is found for estimating the ratio of effective breeding size to the census size, $N_{\rm b}/N$, in \cite{Akita_2020}). 

To estimate the contemporary migration number or migration rate, our simulation-based results provide guidance for a sample size to ensure the required accuracy and precision, especially if the order of the number of migrant parents and parental sizes are approximately known ({\bf Table S1} in {\bf Supporting Information}). For example, when $m=0.1$ and $N_1=N_2=10^3$, sampling 10\% and 20\% parents and an equal number of offspring in two populations leads 70\% and 39\% CV of ${\hat M_5}$, respectively (case of invasive sampling). Even if there is no information about these numbers, $1/\sqrt{H_{\rm HS, bet}}$ (or $1/\sqrt{H_{\rm PO, bet}}$) provides an approximate lower bound on the CV, which can be used as an indicator of the precision of ${\hat M}$ or ${\hat m}$. Furthermore, the condition that $n>\sqrt{N_2}$ is also used as rule of thumb especially when planning a research project ($n = n_{\rm O, 1}=n_{\rm O, 2}=n_{\rm P, 2}$, see also Equations~\ref{RS-1} and \ref{RS-2}). It should be noted that the guidance is to estimate the information about one-way migration between two populations; thus, estimating two-way migrations among two and three populations requires twice and three times the number of samples, respectively, compared to our guidance for a sample size. 

There may be several cases where it is desirable to apply the proposed estimator. The first case is where there is a large number of migrants between populations, e.g., $M>100$, which vanishes the population differentiation and thus hampers the detection of $M$ population genetics method if $M$ is interpreted as effective migration number. Information on movement between populations is essential for assessing population dynamics in the context of conservation and management, even if the migrant number is so large that it cannot be genetically assigned to two populations. The second is the case of the application to genetic monitoring conducted every reproductive timing (e.g., annually). This is because proposed estimators have information about contemporary migration that is explicitly specified the timing (e.g., year), thus the time series data for migration may reflect environmental changes, which provides insights into the underlying ecological processes. In addition, HS pairs found within the genetic population provides $N_{\rm b}$ \cite[]{wang2009new,WAPLES_2011}, which is also used for assessing a genetic health. The third is the case is to develop integrated models that combines several sources of data into a single analysis \cite[]{MAUNDER201361}. One example of this direction is to provide additional information about migration to ongoing CKMR projects for stock assessment in fisheries, which utilizes kinship pairs for simultaneous estimation of population parameters \cite[e.g.,][]{Bravington_2016,Hillary_2018,https://doi.org/10.1002/ece3.6296,ecolevol2021p,10.1093/icesjms/fsac002}. If population structure is hypothetically defined, accumulated data in these projects are readily connected to our theory via likelihood methods. The forth case is where sampling adults is difficult due to conservation purpose or other reasons instead sampling offspring is relatively easy. In this case, although $N_2$ (and $N_1$) need to be given externally for the usage of $\hat m_1$ (and $\hat M_1$), there is presently no other method for estimating adult movement especially when two populations cannot be genetically distinct, indicating that proposed method has the potential to greatly widen the scope of population monitoring. 

Finally, we note some caveats in applying proposed method. Our theory for developing estimators assumes that kinships are detected without any error. There are many algorithms to detect kinship pairs from single nucleotide polymorphisms (SNPs) or short tandem repeats (STRs) \cite[e.g., ][]{Wang_2009Genetics, Huisman_2017}, although a HS pair requires many more DNA markers than a PO pair. In addition, iteroprous species potentially has several kinship types such as halfsib-uncle/nephew or halfsib-cousin, which are expected to appear frequently and should be correctly distinguished from HS pairs. It is desirable to estimate in advance how many markers are needed for kinship detections in target populations associated with simulation of a pedigree reconstruction including distant kinships (e.g., \cite{Anderson_2020}). Our proposed estimators are limited to detect parental movements on a period between breeding seasons in given populations. Thus, the estimation of fine-scale spatial-temporal movements, which is available to integrate multiple type of data \cite[]{Thorson_2021}, goes beyond the scope of this paper, although population dynamics models with a coarser spatial resolution than the spatial scale for environmental layers are often used in the assessment models, which is the target of application of the proposed method. Our proposed estimators require the pre-specification of a population structure. Kinship relationships with information of a sample location potentially provide the chance to explore a plausible population structure via estimating migration number or rate between hypothetical populations, which contributes the determination of a management unit. In addition, the current theory behind the estimators does not assume desynchronized reproduction within a population (e.g., skip spawning, see also \cite{https://doi.org/10.1111/faf.12615}), correlation between individual mobility and fertility and non-random sampling of parents (e.g., mothers with a large number of offspring are likely to be sampled). These perspectives of the study will be conducted in the future.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{ACKNOWLEDGMENTS}
The author thanks N. Shinohara, N. Suzuki and Y. Tsukahara for fruitful discussions. The author also thanks W.M. Iwasaki for helping me in coding C{++}. This work was supported by JSPS KAKENHI Grant Number 19K06862 and 20H03012.

\section*{CONFLICT OF INTEREST}
The author declares no conflict of interest.

\section*{AUTHOR CONTRIBUTIONS}
{\bf Tetsuya Akita:} Conceptualization (lead); formal analysis (lead); funding acquisition (lead); methodology (lead); writing--original draft (lead); writing--review and editing (lead).
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{DATA ACCESSIBILITY}
No datasets were generated or analyzed in this study.

\bibliographystyle{apacite}
\bibliography{akita}

\clearpage

\renewcommand{\arraystretch}{0.6}
\begin{table}[tb]
%\centering
   \caption[]{Table 1: The list of mathematical symbols employed in the main text}
    \textbf {}\\[-4mm]
    \begin{tabular}{llc} \hline
       & & \\
	$n_{\rm P,1}, n_{\rm P, 2}$			& Sampled number of parents from the population 1 and 2\\ 
		                						& \\
	$n_{\rm O, 1}, n_{\rm O, 2}$			& Sampled number of offspring from the population 1 and 2\\ 
		                						& \\
	$N_{1}, N_{2}$						& Number of parents in the population 1 and 2 when sampled offspring are born\\
		                						& \\
	$M$								& Number of survived migrants of parents from the population 1 to population 2\\
		                						& \\
	$m$								& Migration rate of parents from the population 1 to population 2, defined by $M/N_1$.\\
		                						& \\
	$r$								& Sex ratio\\
		                						& \\
	$\pi_{\rm PO,1}, \pi_{\rm PO,2}$		& Probability that a randomly selected pair (parent and offspring) \\
	                							& shares a parent-offspring relationship within the population 1 and 2\\
									& \\
	$\pi_{\rm PO, bet}$					& Probability that a randomly selected pair (parent and offspring) \\
	                							& shares a parent-offspring relationship between the population 1 and 2\\
									& \\
	$\pi_{\rm HS, bet}$					& Probability that a randomly selected pair (two offspring) \\
	                							& shares a half-sibling relationship between the population 1 and 2\\
					                			& \\
	$\lambda_{i,1}, \lambda_{j,2}$			& Expected number of surviving offspring of parent $i$ and $j$ at sampling in the population 1 and 2\\
		                						& \\
	$\lambda_{M,l,1}, \lambda_{M,l,2}$		& Expected number of surviving offspring of migrant $l$ at sampling in the population 1 and 2\\
		                						& \\
	$k_{i,1}, k_{j,2}$					& Number of surviving offspring born to parent $i$ and $j$ in the population 1 and 2\\ 
	                							& \\
	$H_{\rm PO, 1}, H_{\rm PO, 2}$		& Number of parent-offspring pairs observed in samples within the population 1 and 2\\ 
	                							& \\
	$H_{\rm PO, bet}$					& Number of parent-offspring pairs observed in samples between the population 1 and 2\\ 
	                							& \\
	$H_{\rm HS, bet}$					& Number of half-sibling pairs observed in samples between the population 1 and 2\\ 
	                							& \\              		
	\hline
    \end{tabular}
    \label{symbols} 
\\Subscription ``1'' and ``2'' indicates the quantity in the population 1 during the first year and in the population 2 during the second year, respectively. 
\end{table}
\renewcommand{\arraystretch}{1}

\clearpage

\renewcommand{\arraystretch}{0.6}
\begin{table}[tb]
\begin{flushleft}
   \caption[]{Table 2: Summary of proposed estimators for required parameters and kinship types}
    \textbf {}\\[-0mm]
    \begin{tabular}{ccccccccc} \hline
       	\\
	Estimator			& $N_1$ 		& $N_2$ 		& $n_{\rm O, 1}$	& $n_{\rm O, 2}$	& $n_{\rm P, 1}$	& $n_{\rm P, 2}$	& Required kinship type\\
	\\
	\hline
	\\
	$\widehat{M_1}$	& given 		& given 		& \checkmark		& \checkmark 		&				&				& HSP\\ 
		                						& \\
	$\widehat{M_2}$	& given		& given		& \checkmark		& 				&				& \checkmark 		& POP\\
		                						& \\
	$\widehat{M_3}$	& given		& given		& \checkmark		&  \checkmark		&				& \checkmark 		& HSP \& POP\\
		                						& \\
	$\widehat{M_4}$	& given		& estimated	& \checkmark		&  \checkmark		&				& \checkmark 		& HSP \& POP\\
		                						& \\
	$\widehat{M_5}$	& estimated	& estimated	& \checkmark		&  \checkmark		& \checkmark		& \checkmark 		& HSP \& POP\\
									& \\
	$\widehat{m_1}$	& ---			& given		& \checkmark		& \checkmark 		&				&				& HSP\\ 
									& \\
	$\widehat{m_2}$	& ---			& given		& \checkmark		& 				&				& \checkmark 		& POP\\					                			& \\
	$\widehat{m_3}$	& ---			& given 		& \checkmark		&  \checkmark		&				& \checkmark 		& HSP \& POP\\
		                						& \\
	$\widehat{m_4}$	& ---			& estimated	& \checkmark		&  \checkmark		& 				& \checkmark 		& HSP \& POP\\
		                						& \\
	\hline
    \end{tabular}
    \\HSP and POP indicates half-sibling pair and parent-offspring pair, respectively. 
    \label{parameter} 
\end{flushleft} 
\end{table}
\renewcommand{\arraystretch}{1}

\clearpage

\newcommand{\figcapa}{(a) Hypothetical populations with $N_{1}=16$, $N_{2}=14$, and $M=6$. Upper and lower square indicates individuals belonging to the population 1 before migration (at sample timing in the first year) and individuals belonging to the population 2 after reproduction (at sample timing in the second year), respectively. The open circles on the left side, right side, and the center represent mothers, fathers, and their offspring, respectively. The thin line denotes parent-offspring relationship. The bold arrow denotes the migration and the symbol x denotes a failure to survive at sampling in the second year. Sampled individuals are labeled with an index number. The number of sampled individuals in this example: $n_{\rm P, 1}=4$, $n_{\rm O, 1}=5$, $n_{\rm P, 2}=6$, $n_{\rm O, 2}=4$; the numbers of kinship pairs: $H_{\rm PO, bet}=1$ (i.e., ``7-10'' pair), $H_{\rm HS, bet}=2$ (i.e., ``8-16'' and ``9-16'' pairs), $H_{\rm PO, 1}=3$ (i.e., ``1-6'', ``3-5'' and ``4-7'' pairs), and $H_{\rm PO, 2}=2$ (i.e., ``11-17'' and ``15-16'' pairs). (b) The phases of the events relevant to this study along the time line.}

\newcommand{\figcapb}{Violin plots showing the distribution of relative bias in our estimator of $M$ for various values sample sizes, parent numbers and migration rate. Filled circles represent the mean values. Sample sizes for parents and offspring are identical (i.e., $n=n_{\rm P,1}=n_{\rm P,2}=n_{\rm O,1}= n_{\rm O,2}$) and parent sizes in the two populations are also identical (i.e., $N=N_1=N_2$), which are indicated in the legend. Migration rate $m$ is specified by $N_1/M$. For the demonstration purposes, the upper side of the distribution is truncated, although the mean values are calculated including the truncated values. }

%%%FIGURE 1%%%
\begin{figure}[!h]
	\begin{center}
		\includegraphics[width=0.7\textwidth]{/Users/akita/Dropbox/research/close_kin/Mig/Figs/cartoon.eps}
		\caption{{\bf FIGURE 1} \figcapa{}}
		\label{cartoon}
	\end{center}
\end{figure}
%%%%%%%%%%%

%%%FIGURE 2%%%
\begin{figure}[!h]
	\begin{center}
		\includegraphics[width=1\textwidth]{/Users/akita/Dropbox/research/close_kin/Mig/Figs/violins.eps}
		\caption{{\bf FIGURE 2} \figcapb{}}
		\label{violins}
	\end{center}
\end{figure}
%%%%%%%%%%%



\clearpage

\section*{APPENDIX 1}
\setcounter{equation}{0}
\renewcommand{\theequation}{A\arabic{equation}}

\section*{Reproductive potential}

\renewcommand{\theequation}{A\arabic{equation}}
Here, we introduce the concept of the reproductive potential of parent $i$ and $j$ in the population 1 and 2, respectively, which are defined as the expected number of surviving offspring at sampling time, denoted by $\lambda_{i,1}$ and $\lambda_{j,2}$ ($i=1, \ldots, N_1$ and $j=1, \ldots, N_2$). The reproductive potential is determined by several factors, including the parent's age, weight, residence time on the spawning ground. It should be noted that the magnitude of this parameter includes information about the survival rate of the offspring, the number of days after egg hatching, and the egg number; this implies that the parameter reflects the sample timing. It should also be noted that the modeling framework does not depend on whether the reproductive potential is heritable or not.

\section*{APPENDIX 2}
\section*{Derivation of Equations~\ref{HS-1} and \ref{PO-1} when reproductive potential is variable among parents}

In the main text, we ignore the variation of reproductive potential among parents (i.e., both $\lambda_{i,1}$ and $\lambda_{j,2}$ are constant) to derive $\pi_{\rm HS,bet}$ and $\pi_{\rm PO,bet}$. Let $k_{i,1}$ and $k_{j,2}$ be the number of surviving offspring of parent $i$ and $j$ at sampling in the population 1 and 2, respectively, and $k_{i,1}$ and $k_{j,2}$ is assumed to follow a kind of discrete distribution (e.g., negative binomial distribution) with mean $\lambda_{i,1}$ and $\lambda_{j,2}$. Without loss of generality, we set the index such that parents with $i=1$ to $M$ in the population 1 and $j=1$ to $M$ in the population 2 are identical migrants; for example, the parent with $i=1$ reproduce $k_{1,1}$ offspring (in the population 1) and then reproduce $k_{1,2}$ offspring (in the population 2) after migration. Giving $k_{i,1}$ and $k_{j,2}$, the conditional probability that two offspring sampled in the population 1 and 2 share an HS relationship is
\begin{align}
\pi_{\rm HS,bet} | _{\boldsymbol{k_1, k_2}} &= \frac{ 2\sum_{i=1}^M k_{i,1} } { \sum_{i=1}^{N_1} k_{i,1} } \frac{ 2\sum_{j=1}^M k_{j,2} } { \sum_{j=1}^{N_2} k_{j,2} } \frac{1}{M}, 
\end{align}
where $\boldsymbol{k_1}=(k_{1,1}, \ldots, k_{M,1}, \ldots, k_{N_1,1})$ and $\boldsymbol{k_2}=(k_{1,2}, \ldots, k_{M,2}, \ldots, k_{N_2,2})$. It should be noted that $k_{i,1}$ and $k_{j,1}$ is a random variable with mean $\lambda_{i,1}$ and $\lambda_{j,2}$, respectively. By taking the expectation over the distribution of offspring number, the conditional probability is approximately given by
\begin{align}
\pi_{\rm HS,bet} | _{\boldsymbol{\lambda_1, \lambda_2}} &= \mathbb{E}[\pi | _{\boldsymbol{k_1, k_2}}] \nonumber\\
&= \frac{4}{M} \mathbb{E}\left[\frac{ \sum_{i=1}^M k_{i,1} } { \sum_{i=1}^{N_1} k_{i,1} } \frac{ \sum_{j=1}^M k_{j,2} } { \sum_{j=1}^{N_2} k_{j,2} } \right]   \nonumber\\
&\approx \frac{4}{M} \frac{ \mathbb{E}\left[ \sum_{i=1}^M k_{i,1} \sum_{j=1}^M k_{j,2} \right] } {\mathbb{E}\left[ \sum_{i=1}^{N_1} k_{i,1} \sum_{j=1}^{N_2} k_{j,2}\right]} \nonumber\\
&= \frac{4}{M} \frac{ \sum_{i=1}^M \lambda_{i,1} \sum_{j=1}^M \lambda_{j,2} } {\sum_{i=1}^{N_1} \lambda_{i,1} \sum_{j=1}^{N_2} \lambda_{j,2}},
\end{align}
where $\boldsymbol{\lambda_1}=(\lambda_{1,1}, \ldots, \lambda_{M,1}, \ldots, \lambda_{N_1,1})$ and $\boldsymbol{\lambda_2}=(\lambda_{1,2}, \ldots, \lambda_{M,2}, \ldots, \lambda_{N_2,2})$. From the second to the third line, we use the approximation that $\mathbb{E}[g1(k)/g2(k)] \approx \mathbb{E}[g_1(k)] / \mathbb{E}[g_2(k)]$. From the third to the forth line, we use the relationship that $\mathbb{E}[k_{i,1}k_{j,2} | _{\lambda_{i,1}, \lambda_{j,2}}] = \mathbb{E}[k_{i,1}|_{\lambda_{i,1}}]\mathbb{E}[k_{j,2}|_{\lambda_{j,2}}]$, implying statistical independence of offspring number before and after parental movement. In other words, the conditional probability does not affected by $\mathbb{V}[k|_\lambda]$. We assume that $\lambda_{i,1}$ and $\lambda_{j,2}$ is also a random variable followed by an arbitral function with mean ${\bar \lambda_{1}}$ and ${\bar \lambda_{2}}$, respectively. By taking the expectation over $\lambda$ and applying a similar approximation, the unconditional probability is given by
\begin{align}
\pi_{\rm HS,bet} &= \mathbb{E}[\pi | _{\boldsymbol{\lambda_1, \lambda_2}}] \nonumber\\ 
&= \frac{4}{M} \mathbb{E}\left[\frac{ \sum_{i=1}^M \lambda_{i,1} } { \sum_{i=1}^{N_1} \lambda_{i,1} } \frac{ \sum_{j=1}^M \lambda_{j,2} } { \sum_{j=1}^{N_2} \lambda_{j,2} } \right]   \nonumber\\
&\approx \frac{4}{M} \frac{ \mathbb{E}\left[ \sum_{i=1}^M \lambda_{i,1} \sum_{j=1}^M \lambda_{j,2} \right] } {\mathbb{E}\left[ \sum_{i=1}^{N_1} \lambda_{i,1} \sum_{j=1}^{N_2} \lambda_{j,2}\right]} \nonumber\\
&= \frac{4}{M} \frac{M^2 {\bar \lambda_1} {\bar \lambda_2}}{N_1 N_2 {\bar \lambda_1} {\bar \lambda_2}}\nonumber\\
&= \frac{4M}{N_{1}N_{2}},
\label{App2}
\end{align}
which provides the same formulation described in Eq.\ref{HS-1}. It should be noted that, from the third to the forth line, we assume that $\lambda_{l,1}$ and $\lambda_{l,2}$ ($l = 1, \ldots, M$) are independent variables (i.e., $\mathbb{E}[\lambda_{l,1}\lambda_{l,2}] = {\bar \lambda_{1}}{\bar \lambda_{2}}$), implying variable reproductive potential of an identical parent before and after migration. 

Next, we derive the probability that offspring sampled in the population 1 and a parent sampled in the population 2 share an PO relationship ($\pi_{\rm PO,bet}$) under the flexible settings of $\lambda$, which is similar to the derivation of $\pi_{\rm HS,bet}$. The conditional probability is given by
\begin{align}
\pi_{\rm PO,bet} | _{\boldsymbol{k_1, k_2}} &= \frac{ 2\sum_{i=1}^M k_{i,1} } { \sum_{i=1}^{N_1} k_{i,1} } \frac{ \sum_{j=1}^M k_{j,2} } { \sum_{j=1}^{N_2} k_{j,2} } \frac{1}{M}.
\end{align}
By taking the expectation over $k$ and $\lambda$, in the same manner as noted above, the unconditional probability is approximately given by
\begin{align}
\pi_{\rm PO,bet} &= \mathbb{E}[\mathbb{E}[\pi | _{\boldsymbol{k_1, k_2}}]] \nonumber\\ 
&\approx  \frac{2M}{N_{1}N_{2}},
\end{align}
which provides the same formulation described in Eq.\ref{PO-1}.

\section*{APPENDIX 3}

\section*{Difference in reproductive potential between migrants and non-migrants}

In the process of the derivation of $\pi_{\rm HS,bet}$ and $\pi_{\rm PO,bet}$ in {\bf APPENDIX 2}, we ignore covariation between migration and reproductive potential. Here, we consider the situation in which migrants have a distinguishable distribution of reproductive potential from non-migrant parents. Let $\lambda_{i,1}$, $\lambda_{j,2}$, $\lambda_{M,l,1}$ and $\lambda_{M,l,2}$ be a reproductive potential of non-migrant in the population 1 and 2 and migrants in the population 1 and 2 with mean ${\bar \lambda_{1}}$, ${\bar \lambda_{2}}$, ${\bar \lambda_{M,1}}$ and ${\bar \lambda_{M,2}}$, respectively. Under this setting, $\pi_{\rm HS,bet}$ can be written by
\begin{align}
\pi_{\rm HS,bet} &\approx \frac{4}{M}  \frac{ \mathbb{E}\left[ \sum_{l=1}^M \lambda_{M,l,1} \sum_{l=1}^M \lambda_{M,l,2} \right]} {\mathbb{E}\left[ \left(\sum_{l=1}^{M} \lambda_{M,l,1} + \sum_{i=M+1}^{N_1} \lambda_{i,1}\right) \left(\sum_{l=1}^{M} \lambda_{M,l,2} + \sum_{j=M+1}^{N_2} \lambda_{j,2}\right) \right] }\nonumber\\ 
&= \frac{4}{M} \frac{ M^2 {\bar \lambda_{M,1}} {\bar \lambda_{M,2}} } { \left( M {\bar \lambda_{M,1}} + (N_1 - M) {\bar \lambda_1} \right) \left( M {\bar \lambda_{M,2}} + (N_2 - M) {\bar \lambda_2} \right)}.
\label{App3}
\end{align}
If ${\bar \lambda_{M,1}} = {\bar \lambda_{1}}$ and ${\bar \lambda_{M,2}} = {\bar \lambda_{2}}$, Eq.~\ref{App3} reduces to Eq.\ref{App2}. This formulation include reproductive potential terms, so that distinct reproductive potential between migrants and non-migrants vanishes the usefulness from proposed HS-based estimators (this is also applied to PO-based estimators proposed in this paper). 

\clearpage

%\begin{comment}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\section*{Supporting Information}

%\setcounter{figure}{0}
%\newcommand{\figcapsa}{Heatmap showing the relative bias of $\widehat{N_{\rm e,m}/N_{\rm m}}$ as a function of both $n_{\rm M}$ and $n_{\rm O}$. The value of the relative bias is indicated in the legend. The value of the combined effect of parental and nonparental variations increases from left to right ($c=1, 10, 20$, and $100$). (a) $N_{\rm m}=1{,}000$, (b) $N_{\rm m}=10{,}000$. }

%\newcommand{\figcapsb}{Heatmap showing the relative bias of $\widehat{N_{\rm e,m}}$ as a function of both $n_{\rm M}$ and $n_{\rm O}$. The value of the relative bias is indicated in the legend. The value of the combined effect of parental and nonparental variations increases from left to right ($c=1, 10, 20$, and $100$). (a) $N_{\rm m}=1{,}000$, (b) $N_{\rm m}=10{,}000$. }

%\newcommand{\figcapsc}{Heatmap showing the relative bias of $\widehat{1/N_{\rm m}}$ as a function of both $n_{\rm M}$ and $n_{\rm O}$. The value of the relative bias is indicated in the legend. The value of the combined effect of parental and nonparental variations increases from left to right ($c=1, 10, 20$, and $100$). (a) $N_{\rm m}=1{,}000$, (b) $N_{\rm m}=10{,}000$. }

%\newcommand{\figcapsd}{Heatmap showing the coefficient of variation of $\widehat{N_{\rm e,m}/N_{\rm m}}$ as a function of both $n_{\rm M}$ and $n_{\rm O}$. The value of the coefficient of variation is indicated in the legend. The value of the combined effect of parental and nonparental variations increases from left to right ($c=1, 10, 20$, and $100$). (a) $N_{\rm m}=1{,}000$, (b) $N_{\rm m}=10{,}000$.}

%\newcommand{\figcapse}{Heatmap showing the coefficient of variation of $\widehat{N_{\rm e,m}}$ as a function of both $n_{\rm M}$ and $n_{\rm O}$. The value of the coefficient of variation is indicated in the legend. The value of the combined effect of parental and nonparental variations increases from left to right ($c=1, 10, 20$, and $100$). (a) $N_{\rm m}=1{,}000$, (b) $N_{\rm m}=10{,}000$.}

%\newcommand{\figcapsf}{Heatmap showing the coefficient of variation of $\widehat{1/N_{\rm m}}$ as a function of both $n_{\rm M}$ and $n_{\rm O}$. The value of the coefficient of variation is indicated in the legend. The value of the combined effect of parental and nonparental variations increases from left to right ($c=1, 10, 20$, and $100$). (a) $N_{\rm m}=1{,}000$, (b) $N_{\rm m}=10{,}000$.}

%%%FIGURE S1%%%
%\begin{figure}[!h]
%	\begin{center}
%		\includegraphics[width=1\textwidth]{figs/NeN_bias.pdf}
%		\caption{{\bf FIGURE S1} \figcapsa{}}
%		\label{NeN_bias}
%	\end{center}
%\end{figure}
%%%%%%%%%%%

%%%FIGURE S2%%%
%\begin{figure}[!h]
%	\begin{center}
%		\includegraphics[width=1\textwidth]{figs/Ne_bias.pdf}
%		\caption{{\bf FIGURE S2} \figcapsb{}}
%		\label{Ne_bias}
%	\end{center}
%\end{figure}
%%%%%%%%%%%

%%%FIGURE S3%%%
%\begin{figure}[!h]
%	\begin{center}
%		\includegraphics[width=1\textwidth]{figs/Ninv_bias.pdf}
%		\caption{{\bf FIGURE S3} \figcapsc{}}
%		\label{Ninv_bias}
%	\end{center}
%\end{figure}
%%%%%%%%%%%

%%%FIGURE S4%%%
%\begin{figure}[!h]
%	\begin{center}
%		\includegraphics[width=1\textwidth]{figs/NeN_cv.pdf}
%		\caption{{\bf FIGURE S4} \figcapsd{}}
%		\label{NeN_cv}
%	\end{center}
%\end{figure}
%%%%%%%%%%%

%%%FIGURE S5%%%
%\begin{figure}[!h]
%	\begin{center}
%		\includegraphics[width=1\textwidth]{figs/Ne_cv.pdf}
%		\caption{{\bf FIGURE S5} \figcapse{}}
%		\label{Ne_cv}
%	\end{center}
%\end{figure}
%%%%%%%%%%%

%%%FIGURE S6%%%
%\begin{figure}[!h]
%	\begin{center}
%		\includegraphics[width=1\textwidth]{figs/Ninv_cv.pdf}
%		\caption{{\bf FIGURE S6} \figcapsf{}}
%		\label{Ninv_cv}
%	\end{center}
%\end{figure}
%%%%%%%%%%%
%\clearpage
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\end{comment}

\end{spacing}
\end{document}